pandas/tests/test_generic.py

```diff
@@ -1365,6 +1365,23 @@ def test_spline(self):
         expected = Series([1., 2., 3., 4., 5., 6., 7.])
         assert_series_equal(result, expected)
 
+    def test_spline_extrapolate(self):
+        tm.skip_if_no_package('scipy', '0.15', 'setting ext on scipy.interpolate.UnivariateSpline')
+        s = Series([1, 2, 3, 4, np.nan, 6, np.nan])
```

Comment:
I think we can leave that for another PR (as there is an issue for this: #10365), if you don't directly know how to solve it